maro/cli/maro_real_time_vis/start_maro_geo_vis.py

```diff
@@ -77,15 +81,12 @@ def start_geo_vis(start: str, experiment_name: str, front_end_port: int, **kwarg
 
         # Start front-end docker container.
         exec_path = os.path.dirname(inspect.getfile(inspect.currentframe()))
-        os.system("docker pull maro2020/geo_front_service")
+        os.system(f"docker pull {GEO_FRONT_SERVICE_DOCKER_IMAGE}")
         os.system("docker stop geo-vis")
         os.system("docker rm geo-vis")
-        if front_end_port is not None:
-            os.system(f"docker run -d -p {front_end_port}:8080 --name geo-vis maro2020/geo_front_service")
-        else:
-            os.system("docker run -d -p 8080:8080 --name geo-vis maro2020/geo_front_service")
+        os.system(f"docker run -d -p {front_end_port if front_end_port is not None else DEFAULT_FRONT_END_PORT}:8080 "
```

Comment:
Is it the key change for issue #352?

Reply:
No. The key change for issue [352](https://github.com/microsoft/maro/issues/352) is the IP related configurations. I modified two settings, in both frontend and backend, to enable the external access. The first one is in `maro/cli/maro_real_time_vis/back_end/vis_app/app.py` and the second one is in `maro_vis` (the backend docker).